cms/templates/base.html

```diff
@@ -84,6 +84,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
+                      display_course_number: "${context_course.display_coursenumber or ''}",
```

Comment:
I know this is a very small amount of logic, but would it be possible to move the "or" component to the view versus having it here in the template?  It's important to keep the templates as lean as possible -- ie, bind to parameters and avoid additional logic when we can.